native/SampleApps/MobileSyncExplorer/MobileSyncExplorer/Classes/ContactListViewController.m

```diff
@@ -142,10 +141,7 @@ - (void)loadView {
 
     // Search header
     self.searchHeader = [[UIView alloc] initWithFrame:CGRectZero];
-    self.searchHeader.backgroundColor = [[self class] colorFromRgbHexValue:kSearchHeaderBackgroundColor];
```

Comment:
It'll default to black and white now instead, this is how it looks
<img width="892" alt="Screen Shot 2019-10-09 at 2 23 08 PM" src="https://user-images.githubusercontent.com/840431/66523031-c3f9af00-eaa3-11e9-96b8-9fddee62bcbf.png">